server/api/dataset.py

```diff
@@ -128,6 +129,10 @@ def ingestDataset(self, params):
                 'Invalid upload folder ID.', 'uploadFolderId')
 
         name = params['name'].strip()
+        owner = params['owner'].strip()
+        if not owner:
```

Comment:
Why aren't we using `self.requireParams` here (and below)? Is it because we want to `strip` the string first, or because we want a more descriptive error message?